### PR TITLE
fix: Change how `remainingCharacters` is initialized.

### PR DESCRIPTION
**Pull request by jeberhardt**
# Summary | Résumé

closes #841 

| Before                                          | After                                        |
| ----------------------------------------------- | -------------------------------------------- |
| image | 
Screen Shot 2022-06-23 at 1 00 43 PM
 |

# Test instructions | Instructions pour tester la modification

- Load [Form 331](https://forms-staging.cdssandbox.xyz/en/id/331) from staging.
- Observe that the message "You have 0 characters left." does not appear.
- Refresh the page several times, checking that the message does not appear.

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [x] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: components/forms/TextArea/TextArea.tsx [hunk 1]
```diff
@@ -28,13 +28,7 @@ export const TextArea = (
...
   const [field, meta, helpers] = useField(props);
 
-  const [remainingCharacters, setRemainingCharacters] = useState(0);
-
-  useEffect(() => {
-    if (maxLength) {
-      setRemainingCharacters(maxLength);
-    }
-  }, []);
+  const [remainingCharacters, setRemainingCharacters] = useState(maxLength ? maxLength : 0);
```
Comment (bryan-robitaille) [inline components/forms/TextArea/TextArea.tsx:31]: Just a note that the nullish coalescing operator (??) can also be used to shorten this logic statement.
`useState(maxLength ?? 0)`

**Review by bryan-robitaille — CHANGES_REQUESTED** [review]
Let's clean up the unsed imports (useEffect and logMessage) and we should be good to go.

File: components/forms/TextArea/TextArea.tsx (hunk 1 above)
Comment (jeberhardt) [reply components/forms/TextArea/TextArea.tsx]: Love it. Didn't know about the `nullish coalescing operator`.

**Review by bryan-robitaille — DISMISSED** [review]

**Review by bryan-robitaille — APPROVED** [review]
👍

**Pull request merged**